playground/frontend/lib/modules/output/components/output.dart

```diff
@@ -70,10 +74,21 @@ class _OutputState extends State<Output> with SingleTickerProviderStateMixin {
   Widget build(BuildContext context) {
     return Column(
       children: [
-        OutputHeader(
-          tabController: tabController,
-          showOutputPlacements: !widget.isEmbedded,
-          showGraph: widget.showGraph,
+        Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            TabHeader(
+              tabController: tabController,
+              tabsWidget: OutputTabs(
+                tabController: tabController,
+                showGraph: widget.showGraph,
+              ),
+            ),
+            const Padding(
+              padding: EdgeInsets.only(right: kXlSpacing),
```

Comment:
A lone right padding seems odd. Why only right? Why only on the second child of `Row`? This also breaks 'single level of abstraction' principle. A better solution is to put a symmetric padding into `OutputPlacements`. Or at least to make it symmetric here.